Assets/Scripts/Interaction Scripts/ThirdPersonView.cs

```diff
@@ -0,0 +1,450 @@
+﻿namespace ISAACS {
+
+    using System.Collections;
+    using System.Collections.Generic;
+    using UnityEngine;
+        using VRTK;
+        using ISAACS_ControllerInput;
+
+
+    /* TODO
+    */
+
+    public class ThirdPersonView : MonoBehaviour
+    {
+        public GameObject LeftController;
+        public GameObject LeftUI;
+        public GameObject RightController;
+        public GameObject RightUI;
+
+        private VRTK_Pointer LeftPointer;
+        private VRTK_BezierPointerRenderer LeftPointerRenderer;
+        private VRTK_Pointer RightPointer;
+        private VRTK_StraightPointerRenderer RightPointerRenderer;
+        private VRTK_UIPointer RightUIPointer;
+
+        public ControllerInput controllerInput; /// Initiate a singleton of the ControllerState class. Assign it to the Controller GameObject.
+
+        private enum ControllerState { IDLE, SCALING, SELECTING_DRONE, SELECTING_SENSOR, AIMING_SELECTOR, SETTING_WAYPOINT_HEIGHT, PLACING_WAYPOINT, MOVING_WAYPOINT, UNDOING, REDOING }
+        private ControllerState controllerState;
+
+        public GameObject Pivot; /// pivot is the center of the table
+        public GameObject World; /// The World GameObject. All its children will be scaled, rotate and move with it.
+
+        private Vector3 WorldScaleInitial; ///originalScale is the original localScale of the world
```

Comment:
Okay I read a bit more.
```suggestion
        private Vector3 WorldScaleDefault; /// Default local scale of the world. It is also the original localScale on app launch.
```